tests/integration/test_calculate_electricity_cost.py

```diff
@@ -48,7 +48,8 @@ async def test_when_calculate_electricity_cost_uses_real_data_then_calculation_r
     latest_date,
     period_from,
     period_to,
-    tariff_code
+    tariff_code,
+    False
```

Comment:
We should either have this as an input variable for the existing unit tests, or new unit tests should be added to cover this new scenario